prov/efa/src/rxr/rxr_env.c

```diff
@@ -206,7 +200,7 @@ void rxr_env_define()
 	fi_param_define(&rxr_prov, "inter_max_medium_message_size", FI_PARAM_INT,
 			"The maximum message size for inter EFA medium message protocol (Default 65536).");
 	fi_param_define(&rxr_prov, "inter_min_read_message_size", FI_PARAM_INT,
-			"The minimum message size for inter EFA read message protocol. If instance support RDMA read, messages whose size is larger than this value will be sent by read message protocol (Default 1048576).");
+			"The minimum message size for inter EFA read message protocol. If instance support RDMA read, messages whose size is larger than this value will be sent by read message protocol.  Used as a toggle to control the switch from EAGER to READ. (Default 1048576).");
```

Comment:
It seems we are missing this env in efa man page: https://github.com/ofiwg/libfabric/blob/main/man/fi_efa.7.md?plain=1. Could u add it?

Reply:
We are missing: 
```
inter_read_segment_size
inter_min_read_write_size
inter_max_gdrcopy_message_size
efa_cq_read_size
readcopy_pool_size
use_device_rdma
tx_min_credits
```

Will add a new commit for it.

Reply:
talked offline... will do this in separate PR.